core/src/main/resources/no_highlight_log4j2.properties

```diff
@@ -0,0 +1,52 @@
+# Default logging setup for ledger
+
+rootLogger.level = ${env:RADIXDLT_LOG_LEVEL:-info}
+rootLogger.type = asyncRoot
+rootLogger.includeLocation = true
+rootLogger.appenderRef.stdout.ref = STDOUT
+rootLogger.appenderRef.file.ref = FILE
+# Uncomment the following for JSON logging, together with the appender
+#rootLogger.appenderRef.json.ref = JSON
+
+# Squelch illegal reflective access operation debug messages from netty
+loggers=netty-platformdependent0-nodebug
+logger.netty-platformdependent0-nodebug.name=io.netty.util.internal.PlatformDependent0
+logger.netty-platformdependent0-nodebug.level = info
+logger.netty-platformdependent0-nodebug.additivity = false
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{ISO8601} [%p/%c{1}/%t%X{self}] - %m%n
```

Comment:
this config removes highlighting and is meant to be used on our nodes so that grafana doesn't show these annoying characters at the beginning and end of each line